### ISSUE TITLE
Behaviour - action on submit: entry select button does not work

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#918]
**Description**
On the behaviour tab, when I select "redirect to entry" the choose button does not do anything. I cannot select an entry.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Both
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.10
- Craft version: 4.0.3 Pro
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
